dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIBStepPm.mo

```diff
@@ -84,6 +84,7 @@ equation
   generatorSynchronous.switchOffSignal2.value = false;
   generatorSynchronous.switchOffSignal3.value = false;
   annotation(
-    experiment(StartTime = 0, StopTime = 30, Tolerance = 0.0001, Interval = 0.015),
+    experiment(StartTime = 0, StopTime = 30, Tolerance = 0.0001, Interval = 0.015,
+    __OpenModelica_commandLineOptions = "--daeMode"),
```

Comment:
The syntax is actually not the good one: it should be experiment(), __OpenModelica_commandLineOptions="--daeMode", ...
I have corrected it on my side but I get exactly the same problem than @cossartque when he tried to simulate the full HVDC test case in DAE mode.

BTW, seeing the PR made me think about also adding other simulation flags such as forcing to use Kinsol, KLU, IDA and noEquidistantGrid. I can't really test now due to the aforementioned problem but will as soon as possible.

In the meanwhile, I propose to put the PR in standby.

Reply:
@rosiereflo apparently there is a problem in the dae mode with external c code include. Here is what we get with dae mode in the includes.h of the test case:
#ifdef __cplusplus
extern "C" {
#endif
"-I/home/guironnetadr/Dynawo/dynawo/dynawo/dynawo/sources/Models/Modelica/Dynawo/Resources/Include"
#ifdef __cplusplus
}
#endif

And what we get with the ode mode:

#ifdef __cplusplus
extern "C" {
#endif
#include "logEvent.h"
#ifdef __cplusplus
}
#endif

I suppose this is on OpenModelica side, no? Could you confirm? I will try also to test with the OpenModelica version included into Dynawo.

Reply:
This is due to a bug into OpenModelica on the Compiler side. The fix is quite simple but the problem is still present into OpenModelica. Until it is fixed, we should comment the daeMode use.